app/src/main/res/layout/loading_fragment.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <!-- TODO: Update blank fragment layout -->
+
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="246dp"
```

Comment:
I am not a big fan of hardcoded dp values in layouts. It makes it hard to reason about and you will have to change a lot of values manually in case of layout changes. Consider using constraints and margins to indirectly define your width and heights of all views, this will also support different device sizes much better.

Reply:
Okay, I didn't think about it

Reply:
Didn't get how to do it. I use constraint view and margins. But still android studio creates width and height when I change it in the design

Reply:
You can use match_parent if you want it to fill the entire available space, or wrap_content if you only want it to stretch depending on the content.

Reply:
oh, okay